### PR TITLE
New package: python3-cheetah3-3.1.0

### PR DESCRIPTION
**Pull request by Chocimier**

### CODE REVIEW

**Comment by maxice8** [comment]
Why conflict ? make alternatives=

**Comment by Chocimier** [comment]
Should /usr/bin/cheetah be installed as /usr/bin/cheetah3, or alternatives handle it automatically?

**Comment by maxice8** [comment]
should be installed as

**Comment by Chocimier** [comment]
Alternatives added

File: srcpkgs/python3-cheetah3/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+pycompile_module="Cheetah"
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+short_desc="Python-powered template engine and code generator"
+maintainer="Piotr Wójcik <chocimier@tlen.org>"
+license="MIT"
+homepage="http://www.cheetahtemplate.org/"
+distfiles="${PYPI_SITE}/C/Cheetah3/Cheetah3-${version}.tar.gz"
+checksum=0c8d1a2494eaad548544cb2c317514d5279db0ed569e34516b787d5e12b41589
+alternatives="python-cheetah:/usr/bin/cheetah:/usr/bin/cheetah3
```
Comment (maxice8) [inline srcpkgs/python3-cheetah3/template]: Second field can be just the basename if they dirname is the same as the last field

see busybox

Comment (Chocimier) [reply srcpkgs/python3-cheetah3/template]: changed

Comment (Chocimier) [reply srcpkgs/python3-cheetah3/template]: ah, wrong

Comment (Chocimier) [reply srcpkgs/python3-cheetah3/template]: fixed

**Pull request merged**